src/components/NotFound/NotFoundComponent.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { StyledNotFound } from 'components/NotFound/NotFoundStyles';
+
+export function NotFound() {
+  return (
+    <StyledNotFound>
+      <div className="not-found">
+        {/* eslint-disable-next-line react/no-unescaped-entities */}
+        <h3>We didn't find any matches</h3>
```

Comment:
With Material UI we should use `<Typography variant="h3">` instead of a usual `<h3>`, see https://material-ui.com/components/typography/

In our [design](https://www.figma.com/file/FxmwqO6LRg7tOVXrZqMMG0/Neural-Notes-Design-v.0.3?node-id=212%3A836) though we don't have `h3`, but we have `h6`:
![image](https://user-images.githubusercontent.com/6686679/115620584-01c4ce00-a2fe-11eb-808e-e1cb26320a67.png)

We have the styles for it defined in [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L34)

So I think it would be best to use `<Typography variant="h6">` here, as we already have proper styles for it: color, font family, font weight, letter spacing, etc.